src/pages/Scan.vue

```diff
@@ -62,7 +65,7 @@ import Store from '../store.js';
 import Profile from '../components/Profile.vue';
 import Queue from '../components/Queue.vue';
 
-const RESULT_TIMER = 2000;
+const RESULT_TIMER = 1000;
```

Comment:
I think we should keep this the same name as the const in Tap.vue as we want to - at some point - share functionality between the two. We'll probably have to change it back then anyway.